Checking the Genuineness of your Skywallet.md

```diff
@@ -0,0 +1,76 @@
+# Checking the Genuineness of your Skywallet
+
+## Checking your Skywallet Shipment Package for Tampering
+
+Once you receive the Skywallet shipment, please check for any physical tampering or damage of any sorts to the package.
+
+Tampering or damage can be anything from a small tear on the package to a package that is torn open.
+
+**Please do not accept a shipment which seems to have tampered in any such ways.**
+
+### An Untampered Shipment:
+
+<A Picture of an untampered shipment before opening>
+
+### A Tampered Shipment:
+
+<Optional picture of a tampered shipment which is not absolutely torn open but a small tear or hole so that the user will understand even such a small tamper can also spoil the device it can also be a general picture rather than a skywallet shipment>
+
+After opening the shipment package, please ensure that the holographic sticker with the Skycoin logo on the Skywallet box is not broken.
+<A Picture of the unbroken holographic seal on the box>
+
+## Checking the Firmware
+
+Skywallet is built to provide maximum security to the user.
+
+However, it is important to check whether your Skywallet is not compromised before using it.
+
+## Step 1 - Connecting Your Skywallet
+
+As the first step, you have to connect the device to your USB port.
+
+<Optional photo Showing the connection of the device to USB port>
+
+## Step 2 - Checking the message on your Skywallet's Display
+
+### Scenario 1
+
+The display screen equipped on your Skywallet displays one of the following three messages when connected via USB port, marking the authenticity of your Skywallet.
```

Comment:
This sentence should be moved before the section  header for Scenario 1, as it is a general statement rather than part of Scenario  one.

Reply:
Also, ..., marking whether your Skywallet is authentic.